### ISSUE TITLE
[BUG] [Gcloud] BlendNet could not create n2-highcpu-x instances

### ISSUE THREAD

**Issue by Raimund58** [state-of-the-art/BlendNet#140]
### Issue description:
BlendNet could not create n2-highcpu-4 instances.

> "The selected machine type (n2-highcpu-4) has a required CPU platform of **cascadelake**. The minimum CPU platform must match this, but was **skylake**.

```
Feb  6 17:37:12 blendnet-d5lfv2-manager blender[2758]: WARN: Unable to create agent blendnet-d5lfv2-agent-0004 due to error: <HttpError 400 when requesting https://compute.googleapis.com/compute/v1/projects/blendnet-303922/zones/us-central1-f/instances?alt=json returned "The selected machine type (n2-highcpu-4) has a required CPU platform of cascadelake. The minimum CPU platform must match this, but was skylake.">
```

### Environment:
* **Application version**: 2.91
* **Blender client version**: 3.7
* **Blender worker version**: 3.7
* **OS**: Windows 10

### Steps to reproduce:
1. Use Gcloud
2. choose n2-highcpu-4
3. no worker is created

### Expected behavior
A working worker

**Comment by rabits** [comment]
Could you please describe what kind of region/zone you are using (`gcloud config list`)?

So in case you want a speed fix - you can remove those lines in your local and try to recreate the Manager.
* Manager: https://github.com/state-of-the-art/BlendNet/blob/v0.3/BlendNet/providers/gcp/__init__.py#L367
* Agent: https://github.com/state-of-the-art/BlendNet/blob/v0.3/BlendNet/providers/gcp/__init__.py#L490

**Comment by Raimund58** [comment]
`gcloud config list`:
```
C:\Users\Raimund Klink>gcloud config list
[accessibility]
screen_reader = False
[compute]
region = us-central1
zone = us-central1-f
[core]
account = **********@gmail.com
disable_usage_reporting = False
project = blendnet-*******

Your active configuration is: [default]

C:\Users\Raimund Klink>
```

**Comment by rabits** [comment]
Ok, looks fine - did the fix I proposed worked for you?

**Comment by Raimund58** [comment]
You are to fast ;)
I'm still testing.

**Comment by Raimund58** [comment]
Yes, after removing the code bits it works :)

**Comment by rabits** [comment]
Now fixed in [v0.3.8](https://github.com/state-of-the-art/BlendNet/releases/tag/v0.3.8)
